examples/map_kerberos.rst

```diff
@@ -0,0 +1,117 @@
+and ran :code:`dwave config create`, your installation should meet these requirements.
+
+Solution Steps
+==============
+
+Section :ref:`solving_problems` describes the process of solving problems on the quantum
+computer in two steps: (1) Formulate the problem as a :term:`binary quadratic model` (BQM)
+and (2) Solve the BQM with a D-wave system or classical :term:`sampler`. In this example, a
+function in Ocean software handles both steps. Our task is mainly to select the sampler used
+to solve the problem.
+
+Formulate the Problem
+=====================
+
+This example uses the `NetworkX <https://networkx.github.io/>`_ *read_adjlist* function
+to read a text file, `usa.adj`, containing the states of the USA and their adjacencies (states
+with a shared border) into a graph. The original map information
+was found here on `write-only blog of Gregg Lind <https://writeonly.wordpress.com/2009/03/20/adjacency-list-of-states-of-the-united-states-us/>`_ and looks like this::
+
+    # Author Gregg Lind
+    # License:  Public Domain.    I would love to hear about any projects you use if it for though!
+    #
+    AK,HI
+    AL,MS,TN,GA,FL
+    AR,MO,TN,MS,LA,TX,OK
+    AZ,CA,NV,UT,CO,NM
+    CA,OR,NV,AZ
+    CO,WY,NE,KS,OK,NM,AZ,UT
+
+    # Snipped here for brevity
+
+You can see in the first non-comment line that the state of Alaska ("AK") has Hawaii
+("HI") as an adjacency and that Alabama ("AL") shares borders with four states.
+
+>>> import networkx as nx
+>>> G = nx.read_adjlist('usa.adj', delimiter = ',')   # doctest: +SKIP
+
+Graph G now represents states as vertices and each state's neighbors as shared edges.
+Ocean's :std:doc:`dwave_networkx <system:index>` can return a
+`minimum vertex coloring <https://en.wikipedia.org/wiki/Graph_coloring>`_ for a graph,
+which assigns a color to the vertices of a graph in a way that no adjacent vertices
+have the same color. Given a graph representing a map and a :term:`sampler`, the
+`min_vertex_coloring` function tries to solve the map coloring problem.
+
+:std:doc:`dwave-hybrid <hybrid:index>` Kerberos is classical-quantum hybrid asynchronous decomposition sampler, which can decompose large problems into smaller pieces that
+it can run both classically (on your local machine) and on the D-Wave system.
+Kerberos finds best samples by running in parallel :std:doc:`tabu search <tabu:index>`,
+:std:doc:`simulated annealing <neal:index>`, and D-Wave subproblem sampling on
+problem variables that have high impact. The only optional parameters set here
+are a maximum number of iterations and number of iterations with no improvement that
+terminates sampling. (See the :ref:`hybrid1` example for more details on configuring
+the classical and quantum workflows.)
+
+>>> import dwave_networkx as dnx
+>>> from hybrid.reference.kerberos import KerberosSampler
+>>> coloring = dnx.min_vertex_coloring(G, sampler=KerberosSampler(), chromatic_ub=4, max_iter=10, convergence=3)
```

Comment:
The CSP problem defined above is *vertex coloring*, the constrained optimization form is *min vertex coloring*. You should therefore consider using `vertex_color` rather than `min_vertex_color`, or change the language above to constrained optimization.

Reply:
Changed to, "Ocean's :std:doc:`dwave_networkx <system:index>` can return a
`minimum vertex coloring <https://en.wikipedia.org/wiki/Graph_coloring>`_ for a graph,
which assigns a color to the vertices of a graph in a way that no adjacent vertices
have the same color, using the minimum number of colors. "